### DISCUSSION TITLE
Allow custom HTML tags to be added for SEO?

### DISCUSSION THREAD

**Discussion by monolithMktg** [cables-gl/cables_docs#475] [Feature Requests] [answered]
It would be awesome to have a 'custom tag' option under DivElement Op where one can define tags like 'section, aside, header, picture, h1 through h6' etc. for SEO purposes. Or atleast allow an Op to loosely write any HTML we want.

Thanks.

**Comment by pandrr** [comment, marked as answer]
you can write any html and put it into a div as text

e.g. using stringeditor op

but yes, a elementAttribute op would be nice, kinda like the cssProperty op...

**Reply by monolithMktg** [reply]
This  -> "but yes, a elementAttribute op would be nice, kinda like the cssProperty op...".
